.formatter.exs

```diff
@@ -1,5 +0,0 @@
-[
-  import_deps: [:ecto, :phoenix],
-  inputs: ["*.{ex,exs}", "priv/*/seeds.exs", "{config,lib,test}/**/*.{ex,exs}"],
-  subdirectories: ["priv/*/migrations"]
-]
```

Comment:
I think this file was delete by error. I've just checked by creating a new Phoenix 1.7 project and the file is still created automatically. I'll add it back and run `mix format`

Reply:
good spot. thanks for restoring. 👌